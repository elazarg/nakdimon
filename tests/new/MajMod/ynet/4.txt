האווירה הַחוֹרְפִּית נִמְשֶׁכֶת. מָחָר (יוֹם ד') יִהְיֶה גשום לִפְרָקִים, בְּעִיקָּר בִּצְפוֹן וּבַמֶּרְכָּז הָאָרֶץ. הַלַּיְלָה יֵירֵד גֶּשֶׁם מְקוֹמִי בְּעִיקָּר בְּמֶרְכַּז הָאָרֶץ, וְיִיתָּכֵן גַּם גֶּשֶׁם מְקוֹמִי קַל בִּצְפוֹן וּבִדְרוֹם. מָחָר ייחלשו הַגְּשָׁמִים ויתמעטו לִקְרַאת עֶרֶב. מַחְבֶּרֶת "מטאו טֵק" נִמְסַר כִּי תָּחוּל יְרִידָה קַלָּה בְּטֶמְפֵּרָטוּרוֹת. אפיק רדוד בַּקַּרְקַע עֲדַיִין מותיר אֶת האטמוספרה באזורנו בִּלְתִּי יַצִּיבָה עִם סִיכּוּי לגשמים מְקוֹמִיִּים, בְּעִיקָּר בִּצְפוֹן וּבַמֶּרְכָּז. הָרוּחַ בַּיָּם הַתִּיכוֹן תִּהְיֶה דְּרוֹמִית עַד דְּרוֹם מערבית, בִּמְהִירוּת 10 עַד 25 קָמָ"שׁ וְהַיָּם עַצְמוֹ יִהְיֶה גַּלֵּי עַד גַּבָּהּ גַּלִּים, בְּגוֹבַהּ 80-150 ס"מ. הַטֶּמְפֵּרָטוּרוֹת החזויות הַלַּיְלָה וּמָחָר: קִרְיַית שְׁמוֹנֶה 8-16, קצרין 7-14, צפת 6-13, טְבֶרְיָה 12-19, נצרת 9-14, חֵיפָה 12-19, תל-אביב 12-18, יְרוּשָׁלַיִם 9-14, אשקלון 12-21, עַיִן גֶּדִי 14-23, בְּאֵר שֶׁבַע 11-19, עֲרָד 10-16, מְצַפֶּה רמון 7-14, אילת 12-23. בְּיוֹם חֲמִישִׁי יִהְיֶה מעונן חֶלְקִית וְיִיתָּכֵן גֶּשֶׁם מְקוֹמִי בְּעִיקָּר בִּצְפוֹן הָאָרֶץ ומרכזה. תָּחוּל עֲלִיָּיה בְּטֶמְפֵּרָטוּרוֹת. יִיתָּכֵן גֶּשֶׁם מְקוֹמִי קַל בִּדְרוֹם. בְּיוֹם שִׁישִּׁי יֵירֵד גֶּשֶׁם לִפְרָקִים, מְלַוֶּוה בְּסוּפוֹת רְעָמִים בּוֹדְדוֹת, בְּעִיקָּר בִּצְפוֹן הָאָרֶץ. ינשבו רוּחוֹת דְּרוֹם מערביות עֵרוּת. בְּמֶרְכַּז וּבִדְרוֹם יִיתָּכֵן אובך וְאַף יִיתָּכְנוּ סוּפוֹת חוֹל מְקוֹמִיּוֹת שיתחוללו בִּדְרוֹם הָאָרֶץ. בְּשַׁבָּת יֵירֵד גֶּשֶׁם לִפְרָקִים מְלַוֶּוה בְּסוּפוֹת רְעָמִים בִּצְפוֹן וּבַמֶּרְכָּז הָאָרֶץ. ינשבו רוּחוֹת עֵרוּת. חֲשָׁשׁ לשיטפונות וְהַצָּפוֹת בִּמְקוֹמוֹת הַמּוֹעֲדִים. תָּחוּל יְרִידָה בְּטֶמְפֵּרָטוּרוֹת. בְּחֶרְמוֹן יִיתָּכֵן שֶׁלֶג.