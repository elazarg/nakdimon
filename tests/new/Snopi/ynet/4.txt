הָאֲוִוירָה הֶחָורְפִּית נִמְשַׁכְתָּ.
מָחָר (יוֹם ד') יִהְיֶה גָּשׁוּם לִפְרָקִים, בְּעִיקָּר בַּצָּפוֹן וּבַמֶּרְכָּז הָאֶרֶץ.
הַלַּיְלָה יירד גֶּשֶׁם מְקוֹמִי בְּעִיקָּר בְּמֶרְכָּז הָאָרֶץ, וייתכן גַּם גֶּשֶׁם מְקוֹמִי קַל בַּצָּפוֹן וּבַדָּרוֹם.
מָחָר ייחלשו הַגְּשָׁמִים ויתמעטו לִקְרַאת עֶרֶב.
מְחַבֶּרֶת "מטאו טק" נִמְסַר כִּי תָּחוּל יְרִידָה קַלָּה בטמפרטורות.
אָפִיק רָדוּד בַּקַּרְקַע עֲדַיִין מוֹתִיר אֶת הָאַטְמוֹסְפֵרָה בַּאֲזוֹרֵנוּ בִּלְתִּי יְצִיבָה עִם סִיכּוּי לִגְשָׁמִים מְקוֹמִיִּים, בְּעִיקָּר בַּצָּפוֹן וּבַמֶּרְכָּז.
הָרוּחַ בַּיָּם הַתִּיכוֹן תִּהְיֶה דְּרוֹמִית עַד דָּרוֹם מֵעַרְבִית, בַּמְּהִירוּת 10 עַד 25 קָמָ"שׁ וְהַיָּם עַצְמוֹ יִהְיֶה גַּלִּי עַד גָּבָה גַּלִּים, בַּגּוֹבַהּ 80-150 ס"מ.
הטמפרטורות החזויות הַלַּיְלָה וּמָחָר: קִרְיַית שֶׁמּוֹנֶה 8-16, קצרין 7-14, צַפְתָּ 6-13, טְבֶרְיָה 12-19, נצרת 9-14, חֵיפָה 12-19, תֵּל-אָבִיב 12-18, יְרוּשָׁלַיִם 9-14, אֲשְׁקֶלוֹן 12-21, עֵין גְּדִי 14-23, בָּאֵר שָׂבֵעַ 11-19, ערד 10-16, מִצְפֶּה רִמּוֹן 7-14, אַיֶּלֶת 12-23.
בְּיוֹם חֲמִישִׁי יִהְיֶה מְעוּנָּן חֶלְקִית וייתכן גֶּשֶׁם מְקוֹמִי בְּעִיקָּר בְּצָפוֹן הָאָרֶץ וּמֶרְכָּזָהּ.
תָּחוּל עֲלִיָּיה בטמפרטורות.
ייתכן גֶּשֶׁם מְקוֹמִי קַל בַּדָּרוֹם.
בְּיוֹם שִׁישִּׁי יירד גֶּשֶׁם לִפְרָקִים, מְלַוֶּוה בְּסוּפוֹת רְעָמִים בּוֹדְדוֹת, בְּעִיקָּר בַּצָּפוּן הָאָרֶץ.
יִנְשְׁבוּ רוּחוֹת דָּרוֹם מֵעַרְבִיּוֹת עֵרוֹת.
בַּמֶּרְכָּז וּבַדָּרוֹם ייתכן אוֹבֶךְ וְאַף ייתכנו סוּפוֹת חוֹל מְקוֹמִיּוֹת שֶׁיִּתְחוֹלְלוּ בִּדְרוֹם הָאָרֶץ.
בַּשַּׁבָּת יירד גֶּשֶׁם לַפְּרָקִים מְלַוֶּוה בַּסּוּפוֹת רְעָמִים בַּצָּפוֹן וּבְמֶרְכָּז הָאָרֶץ.
יִנְשְׁבוּ רוּחוֹת עֵרוֹת.
חָשַׁשׁ לְשִׁיטְּפוֹנוֹת וַהֲצָפוֹת בַּמְּקוֹמוֹת הַמּוֹעֲדִים.
תָּחוּל יְרִידָה בטמפרטורות.
בחרמון ייתכן שֶׁלֶג.