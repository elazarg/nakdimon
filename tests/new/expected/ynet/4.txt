הָאֲוִוירָה הַחוֹרְפִּית נִמְשֶׁכֶת.
מָחָר (יוֹם ד') יִהְיֶה גָּשׁוּם לִפְרָקִים, בְּעִיקָּר בַּצָּפוֹן וּבְמֶרְכַּז הָאָרֶץ.
הַלַּיְלָה יֵירֵד גֶּשֶׁם מְקוֹמִי בְּעִיקָּר בְּמֶרְכַּז הָאָרֶץ, וְיִיתָּכֵן גַּם גֶּשֶׁם מְקוֹמִי קַל בַּצָּפוֹן וּבַדָּרוֹם.
מָחָר יֵיחָלְשׁוּ הַגְּשָׁמִים וְיִתְמַעֲטוּ לִקְרַאת עֶרֶב.
מֵחֶבְרַת "מֶטֵאוֹ טֵק" נִמְסַר כִּי תָּחוּל יְרִידָה קַלָּה בַּטֶּמְפֵּרָטוּרוֹת.
אָפִיק רָדוּד בַּקַּרְקַע עֲדַיִין מוֹתִיר אֶת הָאַטְמוֹסְפֵרָה בַּאֲזוֹרֵנוּ בִּלְתִּי יַצִּיבָה עִם סִיכּוּי לִגְשָׁמִים מְקוֹמִיִּים, בְּעִיקָּר בַּצָּפוֹן וּבַמֶּרְכָּז.
הָרוּחַ בַּיָּם הַתִּיכוֹן תִּהְיֶה דְּרוֹמִית עַד דְּרוֹם מַעֲרָבִית, בִּמְהִירוּת 10 עַד 25 קָמָ"שׁ וְהַיָּם עַצְמוֹ יִהְיֶה גַּלִּי עַד גְּבַהּ גַּלִּים, בְּגוֹבַהּ 80-150 ס"מ.
הַטֶּמְפֵּרָטוּרוֹת הַחֲזוּיוֹת הַלַּיְלָה וּמָחָר: קִרְיַית שְׁמוֹנָה 8-16, קַצְרִין 7-14, צְפַת 6-13, טְבֶרְיָה 12-19, נָצְרַת 9-14, חֵיפָה 12-19, תֵּל-אָבִיב 12-18, יְרוּשָׁלַיִם 9-14, אַשְׁקְלוֹן 12-21, עֵין גֶּדִי 14-23, בְּאֵר שֶׁבַע 11-19, עֲרָד 10-16, מִצְפֵּה רָמוֹן 7-14, אֵילַת 12-23.
בְּיוֹם חֲמִישִׁי יִהְיֶה מְעוּנָּן חֶלְקִית וְיִיתָּכֵן גֶּשֶׁם מְקוֹמִי בְּעִיקָּר בִּצְפוֹן הָאָרֶץ וּמֶרְכָּזָהּ.
תָּחוּל עֲלִיָּיה בְּטֶמְפֵּרָטוּרוֹת.
יִיתָּכֵן גֶּשֶׁם מְקוֹמִי קַל בַּדָּרוֹם.
בְּיוֹם שִׁישִּׁי יֵירֵד גֶּשֶׁם לִפְרָקִים, מְלוּוֶּה בְּסוּפוֹת רְעָמִים בּוֹדְדוֹת, בְּעִיקָּר בִּצְפוֹן הָאָרֶץ.
יְנַשְּׁבוּ רוּחוֹת דְּרוֹם מַעֲרָבִיּוֹת עֵרוֹת.
בַּמֶּרְכָּז וּבַדָּרוֹם יִיתָּכֵן אוֹבֶךְ וְאַף יִיתָּכְנוּ סוּפוֹת חוֹל מְקוֹמִיּוֹת שֶׁיִּתְחוֹלְלוּ בִּדְרוֹם הָאָרֶץ.
בְּשַׁבָּת יֵירֵד גֶּשֶׁם לִפְרָקִים מְלוּוֶּה בְּסוּפוֹת רְעָמִים בַּצָּפוֹן וּבְמֶרְכַּז הָאָרֶץ.
יְנַשְּׁבוּ רוּחוֹת עֵרוֹת.
חָשַׁשׁ לְשִׁיטְפוֹנוֹת וַהֲצָפוֹת בַּמְּקוֹמוֹת הַמּוּעָדִים.
תָּחוּל יְרִידָה בַּטֶּמְפֵּרָטוּרוֹת.
בַּחֶרְמוֹן יִיתָּכֵן שֶׁלֶג.